spec/requests/restaurants_spec.rb

```diff
@@ -0,0 +1,179 @@
+require 'rails_helper'
+
+RSpec.describe 'Restaurants', type: :request do
+  MAX_RECORD_COUNT = 100
+  RESTAURANT_ATTRIBUTES = %w(id name rating cuisine accept10bis max_delivery_time_min
```

Comment:
`max_delivery_time_min`, naming 😱 
it took me 3 minutes to understand that `min` == minutes, and not minimum

Reply:
and are we sure about the field names? if we have `max_delivery_time_min`, `created_at` and `updated_at` I would expect `accept10bis` to be `accept_10bis`

also, don't think we care about `created_at` and `updated_at`. the fronted need this data?

Reply:
removed the timestamps